runtime/integration-tests/pendulum/src/tests.rs

```diff
@@ -0,0 +1,350 @@
+use crate::{polkadot_test_net::*, setup::*};
+use frame_support::{
+	assert_ok,
+	traits::{fungible::Mutate, fungibles::Inspect, Currency},
+};
+use pendulum_runtime::{Balances, PendulumCurrencyId, RuntimeOrigin, Tokens, XTokens};
+use sp_runtime::{traits::AccountIdConversion, MultiAddress};
+use xcm::latest::{Junction, Junction::*, Junctions::*, MultiLocation, NetworkId, WeightLimit};
+use xcm_emulator::{Junctions, TestExt};
+
+use polkadot_core_primitives::{AccountId, Balance};
+use polkadot_parachain::primitives::Sibling;
+
+const DOT_FEE: Balance = 3200000000;
+const ASSET_ID: u32 = 1984; //Real USDT Asset ID from Statemint
+const INCORRECT_ASSET_ID: u32 = 0;
+pub const UNIT: Balance = 1_000_000_000_000;
+pub const TEN: Balance = 10_000_000_000_000;
+const FEE: u128 = 421434140;
+
+#[test]
+fn transfer_polkadot_from_relay_chain_to_pendulum() {
+	MockNet::reset();
+
+	let transfer_amount: Balance = dot(20);
+	let mut orml_tokens_before = 0;
+	PendulumParachain::execute_with(|| {
+		orml_tokens_before = pendulum_runtime::Tokens::balance(
+			pendulum_runtime::PendulumCurrencyId::XCM(0),
+			&ALICE.into(),
+		);
+	});
+
+	Relay::execute_with(|| {
+		assert_ok!(polkadot_runtime::XcmPallet::reserve_transfer_assets(
+			polkadot_runtime::RuntimeOrigin::signed(ALICE.into()),
+			Box::new(Parachain(2094).into().into()),
```

Comment:
Does it need to be `into().into()` or is it a typo and one `into()` is enough?

Reply:
Also, I can see [here](https://github.com/paritytech/polkadot/blob/645723987cf9662244be8faf4e9b63e8b9a1b3a3/xcm/xcm-simulator/example/src/lib.rs#L210) in the example that they wrapped it with the `X1()` junction. Shouldn't we do this as well?

Reply:
Okay I see. If we don't need it, you can also remove it again. Or also add it to the [AccountId](https://github.com/paritytech/polkadot/blob/645723987cf9662244be8faf4e9b63e8b9a1b3a3/xcm/xcm-simulator/example/src/lib.rs#L211) to make it consistent 😅

Reply:
Done.